ui/lib/kv/addon/components/kv-page-header.hbs

```diff
@@ -0,0 +1,36 @@
+<PageHeader as |p|>
+  <p.top>
+    <Page::Breadcrumbs @breadcrumbs={{@breadcrumbs}} />
+  </p.top>
+  <p.levelLeft>
+    <h1 class="title is-3" data-test-header-title>
+      {{#if @isEngineView}}
+        <Icon @name="kv" @size="24" class="has-text-grey-light" />
+        {{@model.backend}}
+        <span class="tag">
+          Version 2
+        </span>
+      {{else}}
+        {{@pageTitle}}
+      {{/if}}
+    </h1>
+  </p.levelLeft>
+</PageHeader>
+
+{{yield to="tabs"}}
```

Comment:
I think it would be nice if this named block was only for the links and the tab container was in here so you don't have to repeat it in each parent:

```hbs
<div class="tabs-container box is-bottomless is-marginless is-paddingless">
  <nav class="tabs" aria-label="kv tabs">
    <ul>
      {{yield to="tabLinks"}}
    </ul>
  </nav>
</div>
```

Reply:
Good point. I'll amend.